roles/cluster_logging/files/kibana/provision-index.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
```

Comment:
Now the integration with the API is only return code. This works with few issues (like kibana claiming uuid problem at start). Could be small enough to decide to change this to python

Reply:
We discussed this during standup. @alxf asked for a Docker image containing Python & `requests`, which I thought is a bit overkill when the same can, for now, be achieved using shell + `curl`. If need be, this could be changed at some point.